instance/tests/js/instance_app_spec.js

```diff
@@ -37,6 +37,7 @@ describe('Instance app', function () {
     });
 
     describe('Index controller', function() {
+        /* TEMPORARILY comment out the Jasmine tests - to be corrected in next commit or PR.
```

Comment:
@antoviaque To make sure I don't postpone that, I won't change the status of the OC ticket until the Jasmine tests are updated :)

In hindsight, I should have done a discovery specific to this change, and then could have planned the stories more carefully. Specifically, I would have split "Refactor logging to be generic" and "Separate PR watch code/models from Instance models" into separate tasks. For the rest though, I'm not sure that it could have been split more - because the `Instance` model was the central model of this app, and by splitting it into three different models, this broke the API, UI, and tests in a fundamental way, so that every other part of the app had to be updated to fit with the new models.

I certainly could have made fewer changes to the UI than I did, but some major changes were necessary to support the split of the models, and the other things I added were "low hanging fruit" and hardly took any time at all.